src/Microsoft.Restier.Core/ApiBuilder.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using Microsoft.Framework.DependencyInjection;
```

Comment:
The only reason to stay on beta8 is EntityFramework.SqlServer, it's not in RC yet, and it's very weird that EF7.SqlServer beta8 can't work with RC1 build of other EF modules. And since our Northwind (on EF7) sample relies on it. I suggest we do this after EF7.SqlServer releases.